src/server/services/vergunningen/vergunningen.ts

```diff
@@ -417,7 +418,7 @@ export function addLinks(
   });
 }
 
-export async function fetchVergunningen<T>(
+async function fetchVergunningen_(
```

Comment:
Is die underscore een foutje geweest? of begrijp ik iets niet van typescript hier

Reply:
Is geen typescript. 

```
export const fetchVergunningen = memoizee(fetchVergunningen_, {
  maxAge: 45 * ONE_SECOND_MS,
});
```

Reply:
noted!